docs/architecture.md

```
@@ -0,0 +1,127 @@
+SDK接口层 (@onekeyfe/core) 
+    ↓
+传输抽象层 (@onekeyfe/hd-transport)
+    ↓
+平台适配层 (WebUSB/BLE/HTTP)
+    ↓
+硬件设备层 (OneKey设备)
+```
+
+## 🏗️ 核心包结构
+
+### API层
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)